src/common/libsubprocess/fbuf.c

```diff
@@ -161,25 +161,6 @@ static void nonfull_transition_check (struct fbuf *fb, bool was_full)
     }
 }
 
-int fbuf_drop (struct fbuf *fb, int len)
```

Comment:
commit message: have to mention adjust a few callers to use fbuf_read over fbuf_peek + fbuf_drop?  Maybe not a huge deal.